tests/fonts.css

```diff
@@ -0,0 +1,210 @@
+/* Academico */
+@font-face {
+  font-family: 'Academico';
+  font-style: normal;
+  font-display: swap;
+  src:
+    local('Academico'),
+    url(../node_modules/@vexflow-fonts/academico/academico.woff2) format('woff2');
+}
+
+@font-face {
+  font-family: 'Academico';
+  font-style: normal;
+  font-display: swap;
+  font-weight: bold;
+  src:
+    local('Academico'),
+    url(../node_modules/@vexflow-fonts/academico/academico-bold.otf) format('woff2');
+}
+
+/* Bravura */
+@font-face {
+  font-family: 'Bravura';
+  font-style: normal;
+  font-display: block;
+  font-weight: 400;
+  src:
+    local('Bravura'),
+    url(../node_modules/@vexflow-fonts/bravura/bravura.woff2) format('woff2');
+}
```

Comment:
`local('Bravura')` asks the browser to use the font installed on the user's operating system, instead of loading from the url().

If your system has these fonts installed, you'll see that the developer tools / network tab never loads the font file.